src/i18n/locales/en-US.json

```
@@ -547,11 +547,13 @@
       "kills_while_ads_and_moving": "{amount} kills while aiming down sights and moving",
       "kills_while_ads_and_strafing": "{amount} kills while aiming down sights and strafing",
       "kills_while_ads": "{amount} kills while aiming down sights",
+      "kills_shortly_after_sprinting_ads":"get {amount} kills while aiming down sights shortly after sprinting",
```

No need for the "get" word in here 😄 
```suggestion
      "kills_shortly_after_sprinting_ads":"{amount} kills while aiming down sights shortly after sprinting",
```